npc_combat.py

```diff
@@ -34,10 +34,12 @@ def get_npc_clash_values():
     return atk, blk, mod
 
 
-def clash(player_atk, player_blk, player, npc_atk, npc_blk, npc_mod, npc):
+def clash(player_atk, player_blk, player_mod, player, npc_atk, npc_blk, npc_mod, npc):
```

Comment:
I started thinking after your idea to refactor this and here's what I came up with:
```
npc_hits_player = npc_atk != player_blk
player_hits_npc = player_atk != npc_blk

if npc_hits_player:
    player.receive_damage(1)
elif player_mod == "Reversal":
    npc.receive_damage(1)
if player_hits_npc:
    npc.receive_damage(1)
elif npc_mod == "Wolf" or npc_mod == "Star":
    player.receive_damage(1)
```

Reply:
If you like this, or some variation of it, feel free to update code, commit, and push.